lib/init.g

```diff
@@ -1015,8 +1015,9 @@ if GAPInfo.CommandLineOptions.norepl then
 elif IsHPCGAP and THREAD_UI() then
   ReadLib("hpc/consoleui.g");
   MULTI_SESSION();
+  PROGRAM_CLEAN_UP();
 else
   SESSION();
+  PROGRAM_CLEAN_UP();
 fi;
 
-PROGRAM_CLEAN_UP();
```

Comment:
Note that this doesn't constitute an objection to this PR as such; but it's IMHO an issue with libgap-api that we ought to deal with. At the very least, record it as an issue, but of course better to just fix it (and of course ideally, get it into GAP 4.11... which seems to keep slipping anyway *sigh*)

Reply:
I will be the first to admit that I'm not entirely sure about what the best course of action would be. This was prompted by me finding that that `PROGRAM_CLEAN_UP()` was being called during libgap initialization, but not finding a way around avoiding it. It may indeed be better to call `GAP_Initialize()` with augmented arguments.

Reply:
Note that this doesn't constitute an objection to this PR as such; but it's IMHO an issue with libgap-api that we ought to deal with. At the very least, record it as an issue, but of course better to just fix it (and of course ideally, get it into GAP 4.11... which seems to keep slipping anyway *sigh*)